src/test/java/org/opensearch/knn/indices/ModelGraveyardTests.java

```diff
@@ -79,6 +81,87 @@ public void testXContentBuilder() throws IOException {
         assertTrue(testModelGraveyard2.contains(testModelId2));
     }
 
+    // Validating {model_ids:[]}
+    public void testXContentBuilder_withoutModelIds() throws IOException {
```

Comment:
see if we can add the expectation of the test in the function name.

and you can remove throws and add @SneakyThrows instead on the function.

Reply:
I don't think `SneakyThrows` are any better than declaring exception in a test method.

Reply:
its cleaner.

Reply:
I don't agree :)

Reply:
Please provide what is the exact reason why you think it is not cleaner? and what is the usage of a exception in method signature for a test case?

Reply:
I don't think
```
@SneakyThrows
public void test {
```
is any better than
```
public void test throws Exception {
```

`@SneakyThrows` is used to throw checked exception without declaring it in a method's throws clause so that the caller doesn't have to catch the checked exception. 

In the test case, if some methods throws checked exception either we have to catch it inside the test or declare it in the test method's throws clause. I think it is okay to have `throws Exception` in every test methods. It does not look bad personally and there is no caller method which needs to catch the exception in this case as well.

Reply:
For now, I'm merging the PR as I need to include these changes in patch release. Please continue the discussion and if required I will make changes in subsequent PR as the discussion is not about any changes in logic.